.github/workflows/create-release.yml

```diff
@@ -0,0 +1,89 @@
+
+jobs:
+  compute-version:
+    uses: ./.github/workflows/gitversion.yml
+    with:
+      prerelease: "false"
+
+  release-flow:
+    needs: compute-version
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          
+      - name: Output nuget version
+        run: |
+          echo Computed nuget version: ${{ needs.compute-version.outputs.package_version }}
+
+      - name: Setup git config
+        run: |
+          # setup the username and email. I tend to use 'GitHub Actions Bot' with no email by default
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          
+      - name: Start release
+        run: |
+          git checkout develop
+          git checkout -b release/${{ needs.compute-version.outputs.package_version }}
+          
+      - name: Commit
+        run: |
+          # Change version number, commit and push
+          sed -i 's#<PackageVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<PackageVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Definitions/Api.Definitions.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Definitions/Api.Definitions.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest/Api.Rest.csproj
+          sed -i 's#<AssemblyVersion>[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]#<AssemblyVersion>${{ needs.compute-version.outputs.package_version }}#g' ./src/Api.Rest/Api.Rest.csproj
+          git add ./src/Api.Definitions/Api.Definitions.csproj
+          git add ./src/Api.Rest.Dtos/Api.Rest.Dtos.csproj
+          git add ./src/Api.Rest/Api.Rest.csproj
+          git commit -m "Raises version number to ${{ needs.compute-version.outputs.package_version }}"
+          
+      - name: Checkout master
+        run: |
+          git checkout master
+          
+      - name: Merge release into master
+        run: |
+          git merge --no-ff release/${{ needs.compute-version.outputs.package_version }}
+          
+      - name: Tag release commit
+        run: |
+          git tag v${{ needs.compute-version.outputs.package_version }}
+           
+      - name: Checkout develop
+        run: |
+          git checkout develop
+          
+      - name: Merge back master into develop
+        run: |
+          git merge --no-ff master
```

Comment:
Think about putting it all into one script using `echo` instead of `- name`.

Reply:
I bundled all commit and merge operations in one step, so its more compact. Creating the release branch and the final pushes are still their own step, since I find this more readable.